ssl/statem/statem_srvr.c

```diff
@@ -4473,7 +4485,14 @@ int tls_construct_cert_status_body(SSL_CONNECTION *s, size_t chainidx, WPACKET *
 
 CON_FUNC_RETURN tls_construct_cert_status(SSL_CONNECTION *s, WPACKET *pkt)
 {
-    if (!tls_construct_cert_status_body(s, 0, pkt)) {
+    OCSP_RESPONSE *resp;
+
+    resp = get_ocsp_response(s, 0);
+
+    if (resp == NULL)
+        return CON_FUNC_DONT_SEND;
+
+    if (resp != NULL && !tls_construct_cert_status_body(s, resp, pkt)) {
```

Comment:
`resp != NULL` is unnecessary now that you check `resp == NULL` right before it.